cmd/cli/kubectl-kyverno/report/report.go

```diff
@@ -13,47 +11,26 @@ import (
 
 func ComputePolicyReportResult(auditWarn bool, engineResponse engineapi.EngineResponse, ruleResponse engineapi.RuleResponse) policyreportv1alpha2.PolicyReportResult {
```

Comment:
Since `engineResponse` already has the `ruleResponse`, why do we need to pass both?
If there is only the `engineResponse` parameter, we can use `EngineResponseToReportResults()` as it is without altering its behavior.

Reply:
So `ComputePolicyReportResult()` takes both `engineResponse` and `ruleResponse`. I wonder why does it take both? As `engineResponse` already has the `policyResponse` which in return has the `ruleResponse`. If we could make this function takes only the `engineResponse`, then we can safely call `EngineResponseToReportResults()` without making any code changes.

This is the current [EngineResponseToReportResults()](https://github.com/kyverno/kyverno/blob/main/pkg/utils/report/results.go#L89) used by reports controller.